### DISCUSSION TITLE
Google软件工程之过程篇 · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#117] [Announcements]
# Google软件工程之过程篇 · 构建我的被动收入

本文是《Software Engineering at Google》的读书笔记，同时会穿插分享我对软件工程的理解。本文主要介绍软件工程的过程，主要包括Code Review、技术文档与自动化测试（单元、集成、E2E）。

https://www.bmpi.dev/dev/software-engineering-at-google/process/

**Comment by ViktorWong** [comment]
收到的士速递的
